great_expectations/execution_engine/sqlalchemy_execution_engine.py

```diff
@@ -480,6 +480,14 @@ def get_domain_records(
         else:
             selectable = data_object.selectable
 
+        """
+        If a custom query is passed, selectable will be TextClause and not formatted
+        as a subquery wrapped in "(subquery) alias". TextClause must first be converted
+        to TextualSelect using sa.columns() before it can be converted to type Subquery
+        """
```

Comment:
Where did this comment come from? I wrote this comment at one point, and then @cdkini was working on a bugfix in this block I believe.

Reply:
@NathanFarmer I copied it from another section in this module -- the description fit perfectly the current purpose (`RuntimeBatchRequest` containing an SQL query in plain text) and the code was needed in this new place, too.  Thanks.